sessionctx/variable/sysvar.go

```diff
@@ -840,7 +840,7 @@ var defaultSysVars = []*SysVar{
 		s.allowMPPExecution = TiDBOptOn(val)
 		return nil
 	}},
-	{Scope: ScopeGlobal | ScopeSession, Name: TiDBEnforceMPPExecution, Type: TypeBool, Value: BoolToOnOff(DefTiDBEnforceMPPExecution), Validation: func(vars *SessionVars, normalizedValue string, originalValue string, scope ScopeFlag) (string, error) {
```

Comment:
Is the origin global scope involved in any release version?

Reply:
If so, we should remove it in `mysql.global_variables`.

Reply:
No, it is imported in https://github.com/pingcap/tidb/pull/25302 this morning.

Reply:
Okay.